tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -269,9 +269,12 @@ user deletes editable accordion section content block
     user clicks button  Remove block  ${block}
     user clicks button  Confirm
 
-user gets meta guidance data file content editor
-    [Arguments]   ${accordion_heading}
+user enters text into meta guidance data file content editor
```

Comment:
Think it'd be good to have the original `user gets meta guidance data file content editor` keyword as well. In `publish_release_and_amend` I've actually used this to check the editor has been populated with the correct text content.